### ISSUE TITLE
Choosing voices by character

### ISSUE THREAD

**Issue by b4zz4** [R3gm/SoniTranslate#33]
An option to choose the voice for each person detected with whisperx to keep people in a video

**Comment by R3gm** [comment]
Hi,
Could you provide more details about the specific functionality you're referring to?

**Comment by b4zz4** [comment]
Hello, amazing program
For example, a movie and choose the voice of each character. 
Whisperx tells us the number of voices in the movie so we can assign a voice for each actor.

**Comment by Slyfoxleo** [comment]
This has already been done; in the transcription, you can change the voices of the actors by number. 
P.S The author makes a very cool translator, many thanks to Roger Condori!

**Comment by b4zz4** [comment]
@Slyfoxleo Sure, but it's quite a job if they're long videos. I thought in a way of changing that voice in the whole movie at once.
I guess it could be the choice: "☑ Edit generated subtitles"

**Comment by R3gm** [comment]
Yes, by using "Edit generated subtitles", you can see the corresponding speaker and change the TTS voice if needed.
